File: C/BEGINNER/1074 Even or Odd.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int numValues, value;
    scanf("%d", &numValues);
    while (numValues != 0)
    {
        scanf("%d", &value);
        if (value == 0)
            printf("NULL\n");
        else if (value % 2 == 0) // EVEN
        {
            if (value > 0) // EVEN POSITIVE
                printf("EVEN POSITIVE\n");
            else if (value < 0) // EVEN NEGATIVE
                printf("EVEN NEGATIVE\n");
        }
        else if (value % 2 != 0) // ODD
        {
            if (value > 0) // ODD POSITIVE
                printf("ODD POSITIVE\n");
            else if (value < 0) // ODD NEGATIVE
                printf("ODD NEGATIVE\n");
        }
        numValues -= 1;
    }
    return 0;
}

File: Python/BEGINNER/1074 Even or Odd.py
n = int(input())
for i in range(n):
    num = int(input())
    if num == 0:
        print("NULL")
    else:
        print("EVEN" if num % 2 == 0 else "ODD", end=' ')
        print("POSITIVE" if num > 0 else "NEGATIVE")
